config.yaml

```diff
@@ -72,12 +72,26 @@
 
 # default_tenant_token:  <VALID_TENANT_TOKEN>
 
+# Private key type - used for JWT signing
+# Defaults to: rsa
+# Supported values: rsa, ed25519
+# Overwrite with environment variable: DEVICEAUTH_SERVER_PRIV_KEY_TYPE
+
+# server_priv_key_type: rsa
+
```

Comment:
This config option seems a bit redundant and could easily be removed by using the `x509` standard library to parse the private key.

Reply:
Something like this?
https://github.com/mendersoftware/deviceauth/pull/674/commits/938be599eaa33eb6eefd5895a86fe7c73ae5fda9